wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,75 @@
-Why am I banned?
-----------------
+# Handling Cheating
 
--   10 silences issued in your userpage.
--   You gave your account to somebody else.
+
+## Why am I banned?
+A user may get banned if they do any of the following:
+-   Continuous misbehaviour regardless of how many infringements they accumulate.
+-   Circumventing a punishment by making a new account.
+-   Sharing their account with other users.
 -   Using multiple accounts.
--   Cheating (see below).
+-   Hacking or cheating (see below).
 
-What is considered cheating?
-----------------------------
 
-As the rules says, any form of manipulation of the client; score hacking, macro-enabled full-combo sequences, and auto-aim are all considered cheating.
+## What are considered cheating?
+The rules say that any form of manipulation of the client such as score hacking, macro-enabled full-combo sequences, auto-aim and anything else that plays the game for the player.
 
-This is a one-strike rule; cheaters will be **permanently banned.**
+This is a one-strike rule. Cheaters will be banned on their first offence for an indefinite amount of time. The team may decide to lift their ban if they successfully appeal.
```

Comment:
don't do bans, we do 'restrictions', big difference

the entire wording of this article pretty much needs to be blanket adjusted to work under that assumption, because banning is very much a definite thing whereas restrictions are more open ended